src/main/java/edu/ie3/datamodel/io/csv/BufferedCsvWriter.java

```diff
@@ -21,7 +21,7 @@
   /** Information on the shape of the file */
   private final CsvFileDefinition fileDefinition;
   /** True, if every entry should be quoted */
-  private final boolean quoted;
+  private final boolean quoted = true;
```

Comment:
With double double quoting JSON string, quoted should always remain true.
How should we handle this?